packages/financial-templates-lib/src/logger/PagerDutyTransport.ts

```diff
@@ -45,8 +46,12 @@ export class PagerDutyTransport extends Transport {
   // Note: info must be any because that's what the base class uses.
   async log(info: any, callback: (error?: unknown) => void): Promise<void> {
     try {
-      // If the message has markdown then add it and the bot-identifier field. Else put the whole info object as a string
-      const logMessage = info.mrkdwn ? info.mrkdwn + `\n${info["bot-identifier"]}` : JSON.stringify(info);
+      // If the message has markdown then add it (with removed anchor text from links that is not supported by PD) and
```

Comment:
Do we still need the PD v1 transport? Is it used anywhere anymore? cc @daywiss

Reply:
Not that I'm aware of. I kept the same logic, only parsed anchor text from links the same as in PDv2 just for the sake of consistency.